README.md

```diff
@@ -120,11 +120,14 @@ options.
 
 Perform an asynchronous glob search.
 
-## glob.sync(pattern, [options])
+## glob.sync(pattern, options)
```

Comment:
`[ ]` means optionality. I think it need.

Reply:
But in order to run this *asynchronously*, you need to include the `options` object.